test/OpenTelemetry.Instrumentation.Http.Tests/RepeatHandler.cs

```diff
@@ -7,11 +7,11 @@
 
 namespace OpenTelemetry.Tests;
 
-public class RetryHandler : DelegatingHandler
+public class RepeatHandler : DelegatingHandler
 {
     private readonly int maxRetries;
 
-    public RetryHandler(HttpMessageHandler innerHandler, int maxRetries)
+    public RepeatHandler(HttpMessageHandler innerHandler, int maxRetries)
```

Comment:
why the name change?

Reply:
Retry - execute again when failed.
As I see here, it is repeating. The results are not checked.